charts/neon-proxy/templates/deployment.yaml

```
@@ -88,6 +88,17 @@ spec:
             - --sql-over-http-pool-opt-in
             - "false"
             {{- end }}
+            {{- if .Values.settings.disableDynamicRateLimiter }}
+            - --disable-dynamic-rate-limiter
+            - "true"
+            {{- else}}
+            - --disable-dynamic-rate-limiter
+            - "false"
+            {{- end }}
+            {{- if .Values.settings.disableDynamicRateLimiter }}
+            - --aimd-max-limit
+            - {{ . }}
```

You need to pass the setting explicitly, since we're not in a `with` block:
```suggestion
            - {{ .Values.settings.aimdMaxLimit }}
```

---

Added a `with` block.

---

Does `aimdMaxLimit` have a default value in values.yaml?
I think for regions that don't specify that it will pass an empty flag to the binary, I'd wrap this in an `if` block